apps/hash-graph/lib/graph/src/shared/identifier.rs

```diff
@@ -153,20 +153,20 @@ impl EdgeEndpoint for EntityIdWithInterval {
 pub trait EdgeEndpointSet: IntoIterator<Item = Self::EdgeEndpoint> {
     type EdgeEndpoint: EdgeEndpoint;
 
-    fn insert(&mut self, target_id: Self::EdgeEndpoint) -> bool;
+    fn insert(&mut self, target_id: Self::EdgeEndpoint);
 }
 
 impl<S: BuildHasher> EdgeEndpointSet for HashSet<OntologyTypeVertexId, S> {
     type EdgeEndpoint = OntologyTypeVertexId;
 
-    fn insert(&mut self, edge_target_id: Self::EdgeEndpoint) -> bool {
-        self.insert(edge_target_id)
+    fn insert(&mut self, edge_target_id: Self::EdgeEndpoint) {
```

Comment:
Keeping the `bool` return is misleading. An interval may be merged away when iterating the set, but we don't know this, yet. We also never rely on this output.